### PR TITLE
Instructions for converting storage.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: site/en/docs/extensions/reference/storage/index.md [hunk 1]
```diff
@@ -15,14 +15,22 @@ The Storage API provides an extension-specific way to persist user data and stat
...
+Even though extensions can use the [`Storage`][mdn-storage] interface (accessible from `window.localStorage`) in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
 
-- The extension's service worker cannot access `Window.localStorage`.
-- Content scripts share the `Window.localStorage` of the host page.
-- Data saved to `Window.localStorage` is lost when the user clears their browsing history.
+- Extension's service worker cannot access `Storage`.
+- Content scripts share storage with the host page.
+- Data saved using the `Storage` interface is lost when the user clears their browsing history.
+
+Even though extension service workers can use `chrome.storage`, if need to move data, from the `Storage` interface to `chrome.storage` you'll need to use an [offscreen document][offscreen-document].
```
Comment (oliverdunk) [inline site/en/docs/extensions/reference/storage/index.md:27]: Starting this with "even though" made me expect something like "you shouldn't use this". Could we re-word maybe to just be like "If you previously used the Storage interface and need to migrate to extension storage..."?

File: site/en/docs/extensions/migrating/to-service-workers/index.md [hunk 2]
```diff
@@ -100,7 +101,16 @@ textEl.select();
...
+Communicate between offscreen documents and extension service workers using [message passing](/docs/extensions/mv3/messaging/).
+
+## Convert localStorage to chrome.storage.local {: #convert-localstorage }
+
+The web platform's [`Storage`](https://developer.mozilla.org/docs/Web/API/Storage) interface (accessible from `window.localStorage`) cannot be used in a background script. You have two options for replacing it.
+
+* Move its calls to an [offscreen document](/docs/extensions/reference/offscreen/).
+* Replace it with calls to the [`chrome.storage.local`](/docs/extensions/reference/storage/#property-local) namespace.
+
+In the latter case, you will still need to handle the conversion in an offscreen document.
```
Comment (oliverdunk) [inline site/en/docs/extensions/migrating/to-service-workers/index.md:113]: Can we elaborate here? I think it's a bit unclear what we're suggesting.

Comment (jpmedley) [inline site/en/docs/extensions/migrating/to-service-workers/index.md:113]:
```suggestion
The web platform's [`Storage`](https://developer.mozilla.org/docs/Web/API/Storage) interface (accessible from `window.localStorage`) cannot be used in a background script. To address this:

1. Move its calls to an [offscreen document](/docs/extensions/reference/offscreen/).
1. (Optional) Replace it with calls to the [`chrome.storage.local`](/docs/extensions/reference/storage/#property-local) namespace. For details, see [the instructions](/docs/extensions/reference/storage/#can-extensions-use-storage).
```

Comment (jpmedley) [reply site/en/docs/extensions/migrating/to-service-workers/index.md:113]: I left the rewrite in a suggestion to make it easier to see.

File: site/en/docs/extensions/reference/storage/index.md (hunk 1 above)
Comment (jpmedley) [inline site/en/docs/extensions/reference/storage/index.md:27]:
```suggestion
Extension service workers can use `chrome.storage`; however, moving data from the `Storage` interface to `chrome.storage` requires an [offscreen document][offscreen-document]. 
```

Comment (jpmedley) [reply site/en/docs/extensions/reference/storage/index.md:27]: Also in a suggestions.

File: site/en/docs/extensions/migrating/to-service-workers/index.md (hunk 2 above)
Comment (jpmedley) [reply site/en/docs/extensions/migrating/to-service-workers/index.md:113]: I have the anchor this links to in my local copy. I'll push it after you OK the language.

File: site/en/docs/extensions/reference/storage/index.md (hunk 1 above)
Comment (oliverdunk) [reply site/en/docs/extensions/reference/storage/index.md:27]: Thinking about this some more, this is a reference page and not a migration one, so I'm actually not sure that we should even have something about offscreen documents - I think it just adds confusion. If we do want it though I think we need to be more explicit about why we're suggesting it:

```suggestion
Extension service workers can use `chrome.storage` and in most cases, this is preferable. If you previously utilized `Storage` in an MV2 extension, and need to migrate data, you can use an [offscreen document][offscreen-document]. 
```

File: site/en/docs/extensions/migrating/to-service-workers/index.md (hunk 2 above)
Comment (oliverdunk) [reply site/en/docs/extensions/migrating/to-service-workers/index.md:113]: This looks good 👍

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Thumbs up'd one change, left an alternative suggestion on the other :)

File: site/en/docs/extensions/reference/storage/index.md (hunk 1 above)
Comment (jpmedley) [inline site/en/docs/extensions/reference/storage/index.md:27]:
```suggestion
```

**Comment by jpmedley** [comment]
I removed the MV2 stuff. If you're done, please LGTM so I can get a copy edit.

**Review by oliverdunk — APPROVED** [review]
Left one comment, otherwise LGTM. Also don't forget to fix the anchor 😉

**Pull request merged**